ports/nordic/common-hal/busio/I2C.c

```diff
@@ -176,6 +159,11 @@ void common_hal_busio_i2c_deinit(busio_i2c_obj_t *self) {
     self->scl_pin_number = NO_PIN;
```

Comment:
another redundant assignment after addition of mark_deinit. I'll stop noting them for now.